src/quo2/components/list_items/account_list_card/view.cljs

```diff
@@ -0,0 +1,41 @@
+(ns quo2.components.list-items.account-list-card.view
+  (:require
+    [quo2.components.avatars.account-avatar.view :as account-avatar]
+    [quo2.components.icon :as icon]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.list-items.account-list-card.style :as style]))
+
+(defn- internal-view
+  [{:keys [theme account-props networks state action on-press]}]
```

Comment:
@OmarBasem, where are the Figma variations showing how multiple networks should be displayed? I couldn't find it in the design system.

What if there are lots of networks for example? It would be important to have this in the DS.

Reply:
Here is the design: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=900%3A230906&mode=design&t=QHaRfOGf0DWM8pSS-1

Reply:
I see the design, but these variations with multiple networks are not in the design system component @OmarBasem https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=26013-281545&mode=design&t=0OMTW8lnbFCjwMUW-0

Also, weirdly enough, in the Figma [link you shared](https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=900%3A230906&mode=design&t=QHaRfOGf0DWM8pSS-1) the layers don't point to the actual `List items > Account list card` component. Something is off 😕 

I think we should ask designers to fix the component with the variations for multiple networks, instead of implementing this capability based on the real screens, otherwise components start to drift from the DS. cc @J-Son89

Reply:
Yep, definitely!